test/src/test.c

```diff
@@ -463,36 +402,42 @@ static void test_TinyStateCallFunction(void) {
     Tiny_DeleteState(state);
 }
 
-static Tiny_Value CallFunc(Tiny_StateThread *thread, const Tiny_Value *args, int count) {
-    Tiny_Value ret = Tiny_CallFunction(
-        thread, Tiny_GetFunctionIndex(thread->state, Tiny_ToString(args[0])), &args[1], count - 1);
+/*
```

Comment:
Any reason for the commented out code?

Reply:
I have commented the code that is not in use